mycroft/skills/main.py

```diff
@@ -203,9 +203,12 @@ def __init__(self, ws):
         self.enclosure = EnclosureAPI(ws)
 
         # Schedule install/update of default skill
+        self.update_interval = Configuration.get()['skills']['update_interval']
+        self.update_interval = int(self.update_interval * MINUTES)
```

Comment:
Should this be `self.update_interval * 60 * MINUTES` since the time is specified as hours in the config?